Jenkinsfile

```diff
@@ -65,7 +64,7 @@ def buildImageAndRunTests(params, String suffix) {
         def testExtraArgs = ['--no-timestamped-test-subdir']
         if (GlobalVars.isTestSuiteJob) {
             testExtraArgs += ['--kyua-tests-files', '/usr/tests/Kyuafile',
-                              '--no-run-cheritest', // only run kyua tests
+                              '--no-run-cheribsdtest', // only run kyua tests
```

Comment:
Looks like that rename was already pushed to cheribsd so the testsuite job won't work until we merge this.

Reply:
Yeah, there was a bit of a circular dependency here, need updated cheribuild to be able to test this PR, but the flag rename breaks the Jenkinsfile here... I guess maybe that means I should've worked out how to have a compatibility alias after all.